winpr/libwinpr/sspi/Negotiate/negotiate.c

```diff
@@ -160,143 +326,544 @@ static BOOL negotiate_get_config(BOOL* kerberos, BOOL* ntlm)
 	return TRUE;
 }
 
-static SECURITY_STATUS SEC_ENTRY negotiate_InitializeSecurityContextW(
-    PCredHandle phCredential, PCtxtHandle phContext, SEC_WCHAR* pszTargetName, ULONG fContextReq,
-    ULONG Reserved1, ULONG TargetDataRep, PSecBufferDesc pInput, ULONG Reserved2,
-    PCtxtHandle phNewContext, PSecBufferDesc pOutput, PULONG pfContextAttr, PTimeStamp ptsExpiry)
+static BOOL negotiate_write_neg_token(PSecBuffer output_buffer, NegToken token)
```

Comment:
Perhaps passing token as `const NegToken *` would be nice (just an idea)

Reply:
Yes you're absolutely right